src/engine/qcommon/cmd.cpp

```diff
@@ -253,12 +264,7 @@ static void Tokenise( const char *text, char *textOut, bool tokens, bool ignoreQ
 
 				while ( *text && *text != '"' )
 				{
-					if ( *text == '\\' && (++text, (*text >= 0 && *text < ' ') ) )
```

Comment:
These two blocks don't do the same thing. The first one is for an unquoted token which can end at a space. The second is for a string literal which should end with a quote.

Reply:
Right. I do not remember all I had in mind at this time, so for now I'll just move the commit out of the PR.